### PR TITLE
FAQ and DOCUMENTATION update

### LINKED ISSUE #690: Make *scratch* Emacs LIsp by default
@syl20bnr  We shouldn't change it to `Text`. I guess it is because you want `*scratch*` to be something to temporary store text.  However, we can always create a new temporary  buffer by entering non-existent buffer name. Otherwise this behaviour should be documented.

### LINKED ISSUE #1206: Document that prefix argument does not work with helm-M-x
SPC-u M-x does not work with helm-M-x, it fails with:
Error: Specifying a prefix arg before calling `helm-M-x`

This is because `helm-M-x` takes the universal argument in the normal way (C-u) after invocation: https://github.com/emacs-helm/helm/issues/696

It might be good to add a sentence or two to this section: https://github.com/syl20bnr/spacemacs/blob/master/doc/DOCUMENTATION.md#universal-argument.

### LINKED ISSUE #1269: Most of the theme colors are broken for me
Here is a screenshot trying to use `base16-ocean` theme

screen shot 2015-04-20 at 00 07 57

Monokai is the only one that is working nicely, how can I fix this? I'm using iTerm2

### LINKED ISSUE #1504: Visual State streams highlighted text into clipboard
Hey so I have an issue where Spacemacs will stream, or consistently copy, the highlighted text into
my system clipboard. I'm using the official Emacs distribution for Mac OSX (not the port) and should be using the latest Spacemacs. I think this should be something configurable but I can't find the setting looking for some help please. :)

### LINKED ISSUE #1742: status bar  can not display correctly, with messy code, when using an ssh cli like putty or xshell
status bar  can not display correctly, with messy code, when using an ssh cli like putty or xshell on windows. when i use an  ssh client on ubuntu, i works fine.so is there a way to fixing this??

### PR DESCRIPTION
**Pull request by StreakyCobra**
~~Close #690~~ *Not working, another solution should be found*
Close #1206
Close #1269
Close #1504
Close #1742
Correct #3839

### CODE REVIEW

File: doc/FAQ.org [hunk 1]
```diff
@@ -102,11 +107,29 @@ configuration:
 
 See [[https://github.com/syl20bnr/spacemacs/issues/3422#issuecomment-148919047][issue #3422]] and [[https://github.com/emacs-helm/helm/issues/1000#issuecomment-119487649][helm issue #1000]] for details.
 
+** Why does =helm-M-x= (~SPC :~) doesn't accept the prefix argument?
```
Comment (TheBB) [inline doc/FAQ.org]: Why does ... not

File: doc/FAQ.org [hunk 2]
```diff
@@ -102,11 +107,29 @@ configuration:
...
+will fail with this message:
+
+#+begin_verse
+Error: Specifying a prefix arg before calling helm-M-x
+#+end_verse
+
+Instead, call =helm-M-x= first, select the command you want to run, and press
+~C-u~ before pressing ~RETURN~. For instance: ~SPC : org-reload C-u RET~
+
+** Why does my color theme not renders correctly in terminal mode?
```
Comment (TheBB) [inline doc/FAQ.org]: render

File: doc/FAQ.org [hunk 3]
```diff
@@ -102,11 +107,29 @@ configuration:
...
+change your terminal color palette. More explanations can be found on
+[[https://github.com/sellout/emacs-color-theme-solarized#important-note-for-terminal-users][emacs-color-theme-solarized webpage]].
+
 * How do I...
 ** Install a package not provided by a layer?
 Spacemacs provides a variable in the =dotspacemacs/layers= function in
 =.spacemacs= called =dotspacemacs-additional-packages=. Just add a package name
-to the list and it will be installed when you restart.
+to the list and it will be installed when you reload your configuration with
+~SPC f e R~, or at the next Spacemacs lunch.
```
Comment (TheBB) [inline doc/FAQ.org]: launch :-D

File: doc/FAQ.org [hunk 4]
```diff
@@ -282,6 +305,28 @@ To restore the sentence delimiter to two spaces, add the following code to the
 (setq sentence-end-double-space t)
 #+end_src
 
+** Change the initial mode of the =*scratch*= buffer?
+To change the initial mode of the =*scratch*= buffer, add the following code to
+the =dotspacemacs/user-config= function of your =.spacemacs=:
```
Comment (TheBB) [inline doc/FAQ.org]: This tip doesn't actually work. By the time `user-config` is run, the scratch buffer is already created.

File: doc/FAQ.org [hunk 5]
```diff
@@ -282,6 +305,28 @@ To restore the sentence delimiter to two spaces, add the following code to the
...
+
+#+begin_src emacs-lisp
+(setq initial-major-mode 'ruby-mode)
+#+end_src
+
+Change =ruby-mode= accordingly to your needs.
+
+** Prevent the visual selection to override my system clipboard?
```
Comment (TheBB) [inline doc/FAQ.org]: to overrode → overriding

File: doc/FAQ.org [hunk 6]
```diff
@@ -282,6 +305,28 @@ To restore the sentence delimiter to two spaces, add the following code to the
...
+On some operating systems, there is only one clipboard for both *copied* and
+*selected* texts. This has for consequence that visual *selection* – which
```
Comment (TheBB) [inline doc/FAQ.org]: has for → has the

File: doc/FAQ.org [hunk 7]
```diff
@@ -282,6 +305,28 @@ To restore the sentence delimiter to two spaces, add the following code to the
...
+should normally be saved to the ~PRIMARY~ clipboard – overrides the ~SYSTEM~
```
Comment (TheBB) [inline doc/FAQ.org]: primary and system aren't keys. Just emphasise the normal way: `/primary/` or `/system/`.

**Comment by TheBB** [comment]
Made some comments. :-)

The scratch buffer initial major mode doesn't seem to work. Did you try it?

File: doc/FAQ.org (hunk 3 above)
Comment (StreakyCobra) [inline doc/FAQ.org]: :joy:

**Comment by TheBB** [comment]
Thanks! Cherry-picked in develop. You can safely delete your branch.

**Pull request closed without merging**